compiler/tools/fpp-to-cpp/test/component/impl/check-cpp

```diff
@@ -7,6 +7,7 @@ export FPRIME_GCC_FLAGS="-I../../fprime"
 warning_flags="
 -Wno-return-type
 -Wno-unused-parameter
+-Wno-abstract-final-class
```

Comment:
Can we put the new flag first (in alphabetical order)?

Reply:
Done!